lib/pengine/common.c

```diff
@@ -158,6 +158,21 @@ static pcmk__cluster_option_t pe_opts[] = {
         "Setting this to false may lead to a \"split-brain\" situation,"
             "potentially leading to data loss and/or service unavailability."
     },
+    {
+        XML_CONFIG_ATTR_PRIORITY_FENCING_DELAY, NULL, "time", NULL,
+        NULL, pcmk__valid_interval_spec,
+        "Enforced fencing delay targeting the lost nodes with the highest total resource priority",
+        "Enforce specified delay for the fencings that are targeting the lost "
+            "nodes with the highest total resource priority in case we don't "
+            "have the majority of the nodes in our cluster partition, so that "
+            "the more significant nodes potentially win any fencing match, "
+            "which is especially meaningful under split-brain of 2-node "
```

Comment:
It would actually be useful in an even split in any cluster with an even number of nodes

Reply:
Apart from 2-node cluster, clusters with higher even number of nodes probably should better be avoided... Without what "two_node: 1" does for 2-node cluster,  if for example a 4-node cluster gets split in the middle, no partition will be quorate and there won't be fencing match .... right?

Reply:
That makes sense.